### PR TITLE
Caching

### PR DESCRIPTION
**Pull request by shikanime**
Un peu moins de VIH avec du cache de wine

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,16 +13,16 @@
 ## Step 2/4: Run image
 
     # Ye, I known `xhost +` is a bad practice but ¯\_(ツ)_/¯
-    xhost +
+    xhost + $(ifconfig en0 | grep inet | awk '$1=="inet" {print $2}')
```
Comment (ImNotAVirus) [inline README.md]: Doesn't work with Linux (tested with Arch)

File: README.md [hunk 2]
```diff
@@ -13,16 +13,16 @@
...
     docker run \
-        -ti \
-        --env DISPLAY=$DISPLAY \
-        --network=host \
+        -it \
+        --env DISPLAY=$(ifconfig en0 | grep inet | awk '$1=="inet" {print $2}'):0 \
         --name nostale \
         --privileged \
+        --rm \
```
Comment (ImNotAVirus) [inline README.md]: Don't rm your image or you will have to reinstall Nostale

File: README.md [hunk 3]
```diff
@@ -13,16 +13,16 @@
...
 
     # Ye, I known `xhost +` is a bad practice but ¯\_(ツ)_/¯
-    xhost +
+    xhost + $(ifconfig en0 | grep inet | awk '$1=="inet" {print $2}')
     docker run \
-        -ti \
-        --env DISPLAY=$DISPLAY \
-        --network=host \
+        -it \
+        --env DISPLAY=$(ifconfig en0 | grep inet | awk '$1=="inet" {print $2}'):0 \
```
Comment (ImNotAVirus) [inline README.md]: Same as before

**Review by ImNotAVirus — CHANGES_REQUESTED** [review]
Some changes nedded (cf. comments)

File: README.md (hunk 2 above)
Comment (shikanime) [reply README.md]: Caching by binding the $HOME/.wine is fine

Comment (ImNotAVirus) [reply README.md]: Oh, you're right but don't cache all `wine` directory. Only the Nostale directory would be better, I think.

Comment (shikanime) [reply README.md]: Nostale need Gecko, Mono and other dependencies to work so we need everything

Comment (ImNotAVirus) [reply README.md]: You can press `cancel` when `wine` asks you to install Gecko and Mono, it will work

Comment (shikanime) [reply README.md]: I cached only nostale on 7e990bf1deb264ad569258bbe7c9d58811550ade

File: start_nostale.sh [hunk 4]
```diff
@@ -0,0 +1,24 @@
+if [[ "$OSTYPE" == "darwin"* ]]; then
```
Comment (ImNotAVirus) [inline start_nostale.sh]: Please, add a Shebang

File: start_nostale.sh [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        -it \
+        --env DISPLAY=$DISPLAY \
+        --network=host \
+        --name nostale \
+        --privileged \
+        --rm \
+        --volume /tmp/.X11-unix/:/tmp/.X11-unix/ \
+        --volume $HOME/.wine:/root/.wine \
+        nostale
+fi
```
Comment (ImNotAVirus) [inline start_nostale.sh]: Add `host -` before `fi`

File: start_nostale.sh [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    xhost +
+    docker run \
+        -it \
+        --env DISPLAY=$DISPLAY \
+        --network=host \
+        --name nostale \
+        --privileged \
+        --rm \
+        --volume /tmp/.X11-unix/:/tmp/.X11-unix/ \
+        --volume $HOME/.wine:/root/.wine \
```
Comment (ImNotAVirus) [inline start_nostale.sh]: Cache only Nostale here

**Review by ImNotAVirus — APPROVED** [review]

**Pull request merged**